tfhe/Cargo.toml

```diff
@@ -56,12 +54,12 @@ concrete-csprng = { version = "0.4.0", path = "../concrete-csprng", features = [
 lazy_static = { version = "1.4.0", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 rayon = { version = "1.5.0" }
-bincode = { version = "1.3.3", optional = true }
+bincode = "1.3.3" 
```

Comment:
looks like there is a space after the version specifier ?

On VS Code I like the even better toml extension for formatting the toml automatically via alt + shift + F

Reply:
Done